charts/gardener/charts/runtime/templates/deployment-apiserver.yaml

```diff
@@ -161,6 +161,9 @@ spec:
         - --kubeconfig=/etc/gardener-apiserver/kubeconfig/kubeconfig
         {{- end }}
         - --secure-port=443
+        {{- if .Values.global.apiserver.config }}
+        - --admission-control-config-file={{ .Values.global.apiserver.config.admissionControlConfigFile }}
```

Comment:
This does not work. The user will specify the content of the file, but this flag expects a path to a file.
You need to create a ConfigMap containing the configuration (see https://github.com/gardener/gardener/blob/master/charts/gardener/charts/runtime/templates/configmap-apiserver-audit-policy.yaml), declare it as volume (see https://github.com/gardener/gardener/blob/master/charts/gardener/charts/runtime/templates/deployment-apiserver.yaml#L238-L242), and then mount it (see https://github.com/gardener/gardener/blob/master/charts/gardener/charts/runtime/templates/deployment-apiserver.yaml#L184-L187).
After that you know the path and can set the flag correctly (see https://github.com/gardener/gardener/blob/master/charts/gardener/charts/runtime/templates/deployment-apiserver.yaml#L100-L102).